src/oss-defog/DefoggerTool.cs

```diff
@@ -54,28 +55,131 @@ internal class DefoggerTool : OSSGadget
         /// </summary>
         private const int INTERESTING_STRINGS_CUTOFF = 24;
 
+        /// <summary>
+        ///     Enum of executable types
+        /// </summary>
+        public enum EXECUTABLE_TYPE
```

Comment:
Any particular reason why these are ALL_CAPS instead of PascalCase?

Reply:
Nope. I copied them over from another project that uses that format for enums. I can change them.

Reply:
Fixed.